keps/0006-stable-kafka-example.md

```diff
@@ -53,3 +65,65 @@ Here are some features and things we should be able to do to consider Kafka prod
 ### Non-Goals
 
 * Any new KUDO features will not be covered by this KEP
+
+### User Stories
+
+#### Install
+
+As a Kafka administrator I want to deploy a Kafka cluster with a single command so I can get started quickly.
+
+#### Cluster Size
+
+As a Kafka administrator I want to deploy Kafka clusters of different sizes so I can size my cluster to the application.
+
+#### Multiple Clusters
+
+As a Kafka administrator I want to deploy multiple Kafka clusters on the same Kubernetes cluster so that I can use my resources efficiently.
+
+#### Zookeeper Dependency
+
+As a Kafka administrator I want a Zookeeper cluster to be automatically deployed when I deploy Kafka so I don't have to learn how to manage it.
+
+#### Readiness Check
+
+As a Kafka administrator I want to have an indication of when my cluster is ready so that know when to start using it.
+
+#### Health Check
+
+As a Kafka administrator I want to have an indication of cluster health so that know when it is degraded.
+
+#### Vertical Scaling
```

Comment:
I would just use `Configuration Update`

KUDO has concept of `Plans` which enable directed workflows of phase and steps, and have customizable strategies for controlling execution and exposing manual decision points for operators. Plans are an implementation detail that enable:

- Initial deployment
- Scaling capacity
- Configuration update and rollback
- Software upgrade, downgrade and rollback
- Recovery of failed instances
- Replacement of failed instances
- Application Maintenance

```Under this user story, all the details implemented for Kafka must be tested```

As a Kafka admin, I should also be able to reconfigure these services post initial deployment.